charts/managed-identity-wallets/values-pre-prod.yaml

```diff
@@ -20,7 +20,7 @@ certificate:
   host: "managed-identity-wallets.pre-prod.demo.catena-x.net"
```

Comment:
Do we still need the pre-prod values yaml file? I understood that those values are not used